PeerioTechnologies/peerio-desktop
# chore: disable GenerateAccountKey animation when dev env

#### Relevant info and issue/PR links 
https://app.clubhouse.io/peerio/story/13614/desktop-disable-ak-animation-on-dev
 
#### Testing instructions  
`GenerateAccountKey` animation should not run when in dev environment. Should still run on prod.

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
